src/index.js

```diff
@@ -0,0 +1,79 @@
+import 'lodash';
+import './style.css';
+import elementChanges from './check.js';
+import icon from './delete.png';
+import removeAll from './remove.js';
+import Add from './add.js';
+
+let ToDoArray = JSON.parse(localStorage.getItem('ToDoArray') || '[]');
+let title = JSON.parse(localStorage.getItem('title'));
+if (!ToDoArray.length) {
+  document.querySelector('.Activity-list').classList.add('hidden');
+} else {
+  document.querySelector('.Activity-list').classList.remove('hidden');
+}
+document.querySelector('#whatToDo').value = title;
+
+function CreateOb({ description, index }) {
+  this.description = description;
+  this.completed = false;
+  this.index = index;
+}
+
+function displayOnScreen(element) {
+  const ul = document.querySelector('.form_');
```

Comment:
- [ ] Kindly use ES6 syntax for your **CreateOb** and **displayOnScreen** functions.